code/src/App.vue

```diff
@@ -1,19 +1,25 @@
 <template>
-  <div>
-    <h1>Hello world</h1>
+  <div id="app">
     <Footer />
   </div>
 </template>
 
 <script>
 import Footer from "../src/components/Footer";
-export default { components: { Footer } };
+
+export default {
+  name: "App",
+  components: { Footer },
+};
 </script>
 
 <style lang="scss">
 @import url("../src/styles/fonts.scss");
-h1 {
-  color: $prueba;
-  font-family: "fontMontserratBold", sans-serif;
+* {
+  margin: 0;
+}
+a {
+  text-decoration: none;
+  color: white;
```

Comment:
It would be great if you could use a scss variable like the one I created in feature/inicio `$main-font-color` so we can try other color combinations if we want to.

Reply:
Thanks for the reminder, I have made the required changes